en/frames_multicopter/holybro_x500v2_pixhawk6c.md

```diff
@@ -0,0 +1,159 @@
+# Holybro X500 V2 + Pixhawk 6C (PX4 Dev Kit)
+
+This topic provides full instructions for building the [Holybro X500 V2 ARF Kit](http://shop.holybro.com/x500-v2-kit_p1288.html), also known as the Holybro PX4 Dev Kit.
+
+![The fully built vehicle with props removed](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/kit_no_props.jpg)
```

Comment:
FYI I shrunk all the images by file size. This from 6Mb to about 300Kb. Not completely lossless, but you won't notice it on a normal computer screen.
I also used the standard markdown reference for images - this is better because it allows the images to be zoomable by clicking them.